src/patch-events.js

```diff
@@ -440,7 +440,7 @@ export function addEventListener(type, fnOrObj, optionsOrCapture) {
   const wrapperFn = function(e) {
     // Support `once` option.
     if (once) {
-      this[utils.SHADY_PREFIX + 'removeEventListener'](type, fnOrObj, nativeEventOptions);
+      this[utils.SHADY_PREFIX + 'removeEventListener'](type, fnOrObj, optionsOrCapture);
```

Comment:
Not following this change...

Reply:
The `nativeEventOptions` should only be passed to native methods. It's important to pass the actual options to the `removeEventListener` so we can find the wrapper function to remove.